robolectric/src/test/java/org/robolectric/shadows/ShadowParcelFileDescriptorTest.java

```diff
@@ -255,13 +255,15 @@ public void testCreatePipeTwice() throws IOException {
 
   @Test
   public void testGetFd_canRead() throws IOException {
-    assumeThat("Windows is an affront to decency.",
```

Comment:
Maybe we should keep this `assumeThat` to ensure this test will not run on Windows. I think this mechanism will not work on Windows.

Reply:
And this PR is used to fix failed test on macOS, not including Windows.

Reply:
I can try it real quick on Windows

Reply:
> And this PR is used to fix failed test on macOS, not including Windows.

@utzcoz I meant to fix it for all platforms. I believe that Java & JVM can do this, but it should be verified.

Reply:
Hmm, on Windows it has an error, and the error is:
```
org.robolectric.shadows.ShadowParcelFileDescriptorTest > testGetFd_canRead[29] FAILED
    expected to be greater than: 0
    but was                    : -1
        at org.robolectric.shadows.ShadowParcelFileDescriptorTest.testGetFd_canRead(ShadowParcelFileDescriptorTest.java:260)
```

Reply:
@hoisie @ZSmallX We can keep this test skipped on Windows, and fix it on macOS firstly with this PR.

Reply:
Thanks @hoisie! OK, @utzcoz I will revert the assertion on Windows platform and keep the rest of changes.